app/models/government.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class Government
+  include ActiveModel::Model
+
+  def self.find(id)
+    all.find { |government| government.id == id }
+  end
+
+  def self.for_date(date)
+    all.find { |government| government.covers?(date) }
+  end
+
+  def self.current
+    for_date(Date.current)
+  end
+
+  def self.all
+    @all ||= YAML.load_file(Rails.root.join("config", "governments.yml"))
+                 .map { |hash| new(hash) }
+  end
+
+  attr_accessor :id, :slug, :name, :start_date, :end_date
+
+  def ==(other)
+    id == other.id
+  end
+
+  def eql?(other)
+    self == other
```

Comment:
I don't suppose we can do `alias_method :==, :eql?` here? (I suspect it triggers a syntax error, but if it works it'll be a cleaner solution!).

Reply:
I'll see if that works, I just robbed this from a class we already had that was similar.